frontend/utils/api.js

```diff
@@ -18,6 +18,29 @@ const formatGroceryItemAdd = (formData) => {
     }
 }
 
+
+
+export const addPrice = (formData) => {
```

Comment:
I think it is better to add comments to this helper function to explain it's functionality and expected outcome.

Reply:
Good idea - we will properly document this function.